homeassistant/components/home_connect/__init__.py

```diff
@@ -138,11 +138,18 @@ async def _async_service_program(call, method):
         """Execute calls to services taking a program."""
         program = call.data[ATTR_PROGRAM]
         device_id = call.data[ATTR_DEVICE_ID]
-        options = {
-            ATTR_KEY: call.data.get(ATTR_KEY),
-            ATTR_VALUE: call.data.get(ATTR_VALUE),
-            ATTR_UNIT: call.data.get(ATTR_UNIT),
-        }
+
+        options = []
+
+        option_key = call.data.get(ATTR_KEY)
+        if option_key is not None:
```

Comment:
This will always be true.

Reply:
It would be if it were required, but [it is not](https://github.com/home-assistant/core/pull/76411#discussion_r945327456).